src/engine/uct/node/mod.rs

```diff
@@ -146,21 +146,38 @@ impl Node {
                 .iter()
                 .map(|&m| Node::new(m, self.config))
                 .collect();
+            if self.children.len() <= (game.size() * game.size() / 10) as usize {
+                let player = game.last_move().color().opposite();
+                if game.winner() == player || !self.config.uct.playout_aftermath { //don't pass if we're losing on the board
```

Comment:
Because the check only applies to sending a pass as a move while behind. We can simulate passes in the tree (for ko purposes).